packages/react/src/ComparisonTable/ComparisonTable.stories.tsx

```diff
@@ -0,0 +1,169 @@
+      },
+      table: {
+        category: 'ComparisonTable'
+      }
+    },
+    variant: {
+      options: [0, 1, 2], // iterator
+      mapping: [undefined, 'default', 'minimal'], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['undefined', 'default', 'minimal']
+      },
+      table: {
+        category: 'ComparisonTable'
+      }
+    },
+    heading: {
+      type: 'string',
+      table: {
+        category: 'ComparisonTable'
+      }
+    },
+    featuredColumn: {
+      type: 'number',
+      defaultValue: 1,
+      table: {
+        category: 'ComparisonTable'
+      }
+    },
+    footnote: {
+      type: 'string',
+      defaultValue:
+        '*** This is a biased overview of capabilities by use case, based on publicly available information as of 2022-05-16.',
+      table: {
+        category: 'ComparisonTable'
+      }
+    }
+  },
+  parameters: {
+    viewport: {
+      viewports: INITIAL_VIEWPORTS
+    }
+  }
+} as ComponentMeta<typeof ComparisonTable>
+
+const Template: ComponentStory<typeof ComparisonTable> = args => (
+  <ComparisonTable featuredColumn={args.featuredColumn} heading={args.heading} variant={args.variant} as={args.as}>
+    <ComparisonTable.Row>
+      <ComparisonTable.Cell>Use case</ComparisonTable.Cell>
+      <ComparisonTable.Cell>GitHub</ComparisonTable.Cell>
+      <ComparisonTable.Cell>Jenkins</ComparisonTable.Cell>
+    </ComparisonTable.Row>
+    <ComparisonTable.Row>
+      <ComparisonTable.Cell>Automation & CI/CD</ComparisonTable.Cell>
+      <ComparisonTable.Cell>
+        <Text as="p" size="300">
+          Comparable native core capabilities
+        </Text>
+        <Text as="p" size="300">
+          <InlineLink href="#">Over 13,000 GitHub Actions are available</InlineLink>
+          &nbsp;in the GitHub Marketplace to automate your development workflow.
+        </Text>
+      </ComparisonTable.Cell>
+      <ComparisonTable.Cell>
+        <Text as="p" size="300">
+          Comparable native capabilities
+        </Text>
+        <Text as="p" size="300">
+          1,800+ community contributed Jenkins plugins <InlineLink href="#">in Jenkins Plugin Marketplace.</InlineLink>
+        </Text>
+      </ComparisonTable.Cell>
+    </ComparisonTable.Row>
+    <ComparisonTable.Row>
+      <ComparisonTable.Cell>Deployment models</ComparisonTable.Cell>
+      <ComparisonTable.Cell>Cloud or self-hosted</ComparisonTable.Cell>
+      <ComparisonTable.Cell>
+        <Text as="p" size="300">
+          Self-hosted only
+        </Text>
+        <Text as="p" size="300">
+          CloudBees is the cloud alternative
+        </Text>
+      </ComparisonTable.Cell>
+    </ComparisonTable.Row>
+    {args.footnote && <ComparisonTable.Footnote>{args.footnote}</ComparisonTable.Footnote>}
+  </ComparisonTable>
+)
+
+export const Playground = Template.bind({})
+Playground.args = {
+  heading: 'GitHub vs Jenkins'
+}
+
+export const PlaygroundMobile = Template.bind({})
```

Comment:
Based on the ADR I thought we were moving away from using Template.bind? Is this the case?

Reply:
I may have misunderstood but I don't think the ADR describes it that way 🤔 

At a minimum, the playground would always need to use Template.bind, as that's how docgen will auto-infer prop types.

For all other stories.. from ADR:

> Feature stories may leverage Storybook's Template.bind() for efficiency, or use syntax that's representative of how a consumer would use the component. For compositional components that render sub-child components, it may be easier to avoid Template.bind()

There's been a few instances in SubdomainNavBar where I wasn't able to use Template.bind, but those are rare in my experience. I think the `minimal` variant for ComparisonTable also uses a fixed story structure due to having different content.

Reply:
@rezrah is correct here! The ADR lists the consequence of using `Template.bind()` as it will currently make the code snippet somewhat useless, but we hope to fix that addon in Storybook instead of _not_ using `Template.bind()`

Reply:
@langermank after our talk it seems like we can use it just knowing that it currently isn't perfect correct?

Reply:
@JoshBowdenConcepts Yup!